articles/architecture-scenarios/index.md

```diff
@@ -1,80 +1,65 @@
 ---
 url: /architecture-scenarios
 title: Architecture Scenarios
+classes: topic-page
 description: Learn about the common architecture scenarios that you will use to solve the authorization and authentication needs of your application.
 ---
 # Architecture Scenarios
 
 This page describes the typical architecture scenarios we have identified when working with customers on implementing Auth0.
 
-Click on any scenario to get more information.
+![Sample Architecture Scenario](/media/articles/architecture-scenarios/mobile-api.png)
 
 ## Application Configurations
 
 These scenarios describe the different type of technology architectures your application may use, and how Auth0 can help for each of those.
```

Comment:
Can you add a couple more sentences here and explain the structure of these scenarios? (that we present an imaginary business case, set goals and requirements, explain the architectural solution, and offer a sample implementation)